venues/MIDL.io/2019/Conference/python/recruit_area_chairs.py

```diff
@@ -0,0 +1,82 @@
+
+5. During the review period, each AC is expected to read and be able to give a score/opinion on their assigned papers (about 10 papers).
+
+6. It is important that each paper receives 3 reviews from the reviewers. In case  there are missing reviews (we hope this number will be minimal this year), the Area Chairs, by informing/help of the Program Chairs, will select last-minute reviewers, each of whom should be able to complete 1-2 reviews in at most 2-3 days. Last minute reviews can be handled by other ACs.
+
+7. The official reviews (i.e. the 3 reviews ) for each paper will be posted publicly and anonymously after all three reviews for each paper are collected. The reviewers will have the option of de-anonymizing their reviews to be posted after the decisions are announced. At this point, the OpenReview system will also be open to receiving public posts/comments, however, those will not be made public until the decisions are announced. They will not be visible to Area Chairs and the Program Chairs, however they will be visible to the Authors during the whole process.
+
+8. The authors will be able to address feedback of the official reviewers in by responding to reviewer comments on the OpenReview system and make changes to the paper, if they would like to. The rebuttal and discussion period will be from January 28, 2019 until February 04, 2019. The author responses will be publicly visible at the OpenReview system.
+
+9. After the rebuttal process ends, each AC will evaluate the 3 received reviews/scores and responses/rebuttal for each of their papers. They will submit  a score and a tentative recommendation for all the papers in their pool. The deadline for AC recommendation upload to the system is February 13, 2019. ACs will have the chance to override the majority voting decision from the reviewers, in special certain cases where they believe that the comments of the reviewers do not match the quantitative scores given to a paper. In that case, AC should provide a justification for their recommendation. We again do not expect that this will happen much or even at all due to the "carefully" constructed reviewer pool.
+
+10. The program chairs will collect all the AC recommendations and will do a final check on the consistency of the reviewer scores + AC scores + AC recommendations. The final decisions will be announced on February 20th, 2019.
+
+In summary, finally, the Program chairs will go through the 4 scores (3 Reviewers + 1 AC review + AC recommendation) to detect any major inconsistencies, if any. In most cases, AC recommendation will be the base resulting decision, however, we will check outlier cases, where for instance 3 reviewers return with negative results and the AC returns with a positive decision and such.
+
+'''
+
+conference.recruit_reviewers(emails = formatted_acs, title = title, message = message, reviewers_name = 'Area_Chairs')
```

Comment:
to remind reviewers/areachairs you can add the parameter: remind = True:

```
conference.recruit_reviewers(title = title, message = message, reviewers_name = 'Area_Chairs', remind = True)
```

Reply:
this will send an email with the same text and subject: 'Reminder' + title to the invited reviewers that haven't responded yet.